project_liberation/settings/base.py

```diff
@@ -143,6 +144,99 @@
 
 USE_TZ = True
 
+
+LOGGING = {
+    'version':                  1,
+    'disable_existing_loggers': False,
+    'formatters':               {
+        'console': {
+            'format':  '%(asctime)s %(levelname)-8s | %(message)s',
+            'datefmt': '%H:%M:%S',
+        },
+    },
+    'filters': {
+        'require_debug_false': {
+            '()': 'django.utils.log.RequireDebugFalse',
+        }
+    },
+    'handlers': {
+        'sentry': {
+            'level': 'ERROR',
+            'class': 'raven.contrib.django.raven_compat.handlers.SentryHandler',
+        },
+        'mail_admins': {
+            'level':   'ERROR',
+            'filters': ['require_debug_false'],
+            'class':   'django.utils.log.AdminEmailHandler',
+        },
+        'console': {
+            'level':     'INFO',
+            'class':     'logging.StreamHandler',
+            'formatter': 'console',
+        },
+    },
+    'loggers': {
+        # NOTE: There are a few important caveats you need to consider when tweaking logging:
+        # - If a message is logged to logger 'a.b' and propagates to logger 'a', only the level of logger
+        #   'a.b' counts. Level of logger 'a' is ignored. Not kidding. See the diagram:
+        #   https://docs.python.org/3/howto/logging.html#logging-flow
+        # - level of logger 'a.b' determines not only what it handles but also what it propagates to parent.
+        # - If a logger has no level, it inherits level from parent. Root logger (the one called '') has level WARNING by default.
+
+        # RULES: Try to stick to the following conventions:
+        # - Don't set level of a logger unless you explicitly want to prevent some messages from being handled or propagated.
```

Comment:
It's fine, the rule is just not 100% precise because I did not want to state the obvious and make it more confusing than it needs to be.

If you want this to be precise you could reword it to say `Don't set level of a logger to anything higher than DEBUG ...` and clarify that setting `DEBUG` is also fine when a different level would be inherited from the parent logger. I didn't think it was necessary.